src/pages/Approvals.tsx

```
@@ -108,12 +108,36 @@ export default function Approvals() {
-        target_type: targetType,
-        status: currentStatus,
-      });
+      // 使用基础的 fetchApprovals，然后在前端进行筛选
+      const allApprovals = await fetchApprovals();
+
+      // 前端筛选逻辑（临时方案，建议后端支持参数）
+      let filteredApprovals = allApprovals;
+
+      // 根据类型筛选
+      if (targetType && targetType !== 'all') {
+        filteredApprovals = filteredApprovals.filter((item: any) =>
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)